diff --git a/src/applications/gqrx/receiver.cpp b/src/applications/gqrx/receiver.cpp
index b79b44c..11f2b73 100644
--- a/src/applications/gqrx/receiver.cpp
+++ b/src/applications/gqrx/receiver.cpp
@@ -319,14 +319,14 @@ double receiver::set_input_rate(double rate)
 
     tb->lock();
     ssa_mode = false;
-    if (src->ioctl("ssa_support", (uint64_t)&max_sps)) {
-        if (rate > max_sps) {
-            ssa_mode = true;
-        }
-    }
+    // if (src->ioctl("ssa_support", (uint64_t)&max_sps)) {
+    //     if (rate > max_sps) {
+    //         ssa_mode = true;
+    //     }
+    // }
 
     if (ssa_mode) {
-        d_input_rate = src->ioctl("set_ssa_sample_rate", (uint64_t)rate);
+        // d_input_rate = src->ioctl("set_ssa_sample_rate", (uint64_t)rate);
 
         if (d_input_rate > 0 ) {
 //            d_input_rate = src->ioctl("get_ssa_config", &num_bin, &halfband);
@@ -335,7 +335,7 @@ double receiver::set_input_rate(double rate)
             iq_fft->set_sweeping_mode(true, num_bin, halfband);
         }
     } else {
-        d_input_rate = src->ioctl("set_ssa_sample_rate", (uint64_t)0);
+        // d_input_rate = src->ioctl("set_ssa_sample_rate", (uint64_t)0);
         iq_fft->set_sweeping_mode(false, 0, 0);
         d_input_rate = src->set_sample_rate(rate);
     }
